portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/antisamy/AntiSamy.testcase

```diff
@@ -61,6 +61,47 @@ definition {
 		}
 	}
 
+	@description = "LPD-25553: AntiSamy Should be disabled without exceptions"
+	@priority = 3
+	test AntiSamyShouldBeDisabledWithoutExceptions {
+		property antisamy.enabled = "true";
```

Comment:
Looks likt this property is used to initialize a config file.  From build.xml:
		<get-testcase-property property.name="antisamy.enabled" />

```
		<if>
			<equals arg1="${antisamy.enabled}" arg2="true" />
			<then>
				<antcall target="prepare-antisamy-config-file" />

				<gradle-execute dir="${project.dir}/modules/apps/portal-security/portal-security-antisamy" task="deploy">
					<arg value="clean" />
				</gradle-execute>
			</then>
		</if>
```